src/app/js/public/search/sagas.js

```diff
@@ -0,0 +1,75 @@
+import { put, select, call, takeEvery } from 'redux-saga/effects';
+
+import { SEARCH, searchSucceed, searchFailed } from './';
+import { fromUser } from '../../sharedSelectors';
+import fetchSaga from '../../lib/sagas/fetchSaga';
+
+const OVERVIEW_TITLE = 1;
+const OVERVIEW_DESCRIPTION = 2;
+
+let overviewFields;
+
+const getOverviewFields = function*() {
```

Comment:
try to use findFieldWithOverviewID from  /src/app/js/fields/selectors.js instead